content/docs/04-clusters/03-edge/02-edge-configuration/35-installer-reference.md

```diff
@@ -106,7 +106,7 @@ The `stylus.site` blocks accept the following parameters.
 | --- | --- |
 | `paletteEndpoint` | The URL endpoint that points to Palette. Example: `api.spectrocloud.com` | 
```

Comment:
Should this say "The API endpoint..."?

Reply:
It should be. I'll ask Karl.

Thanks for the review, Rita.